gnocchiclient/benchmark.py

```diff
@@ -12,21 +12,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 import argparse
+from concurrent import futures
 import datetime
 import functools
 import logging
 import math
 import random
 import time
 import types
+import warnings
 
 from cliff import show
-import futurist
-from oslo_utils import timeutils
 import six.moves
 
+from gnocchi import utils
```

Comment:
this seems like a bit much to be pulling in entire gnocchi lib just for client no?